scripts/srt-dev.lua

```diff
@@ -0,0 +1,887 @@
+fields.none = ProtoField.none("srt_dev.none", "none", base.NONE)
+
+-- Data packet fileds
+fields.data_flag_info_tree = ProtoField.uint8("srt_dev.data_flag_info_tree", "Data Flag Info", base.HEX)
+local FF_state_select = {
+	[0] = "[Middle packet]",
+	[1] = "[Last packet]",
+	[2] = "[First packet]",
+	[3] = "[Single packet]"
+}
+fields.FF_state = ProtoField.uint8("srt_dev.FF_state", "FF state", base.HEX, FF_state_select, 0xC0)
+local O_state_select = {
+	[0] = "[Data delivered unordered]",
+	[1] = "[Data delivered in order]"
+}
+fields.O_state = ProtoField.uint8("srt_dev.O_state", "O state", base.HEX, O_state_select, 0x20)
+local KK_state_select = {
+	[0] = "[Not encrypted]",
+	[1] = "[Data encrypted with even key]",
+	[2] = "[Data encrypted with odd key]"
+}
+fields.KK_state = ProtoField.uint8("srt_dev.KK_state", "KK state", base.HEX, KK_state_select, 0x18)
+local R_state_select = {
+	[0] = "[Not Set]",
+	[1] = "[Retransmit Packet]"
+}
+fields.R_state = ProtoField.uint8("srt_dev.R_state", "R state", base.HEX, R_state_select, 0x04)
+fields.seq_num = ProtoField.uint32("srt_dev.seq_num", "Sequence Number", base.DEC)
+fields.msg_num = ProtoField.uint32("srt_dev.msg_num", "Message Number", base.DEC)--, nil, 0x3FFFFFF)
+
+-- control packet fileds
+local msg_type_select = {
+	[0] = "[HANDSHAKE]",
+	[1] = "[KEEPALIVE]",
+	[2] = "[ACK]",
+	[3] = "[NAK(Loss Report)]",
+	[4] = "[Congestion Warning]",
+	[5] = "[Shutdown]",
+	[6] = "[ACKACK]",
+	[7] = "[Drop Request]",
+	[8] = "[Peer Error]",
+	[0x7FFF] = "[Message Extension Type]"
+}
+fields.msg_type = ProtoField.uint16("srt_dev.msg_type", "Message Type", base.HEX, msg_type_select, 0x7FFF)
+fields.msg_ext_type = ProtoField.uint16("srt_dev.msg_ext_type", "Message Extented Type", base.DEC)
+
+-- Handshake packet fields
+fields.UDT_version = ProtoField.uint32("srt_dev.UDT_version", "UDT Version", base.DEC)
+fields.sock_type = ProtoField.uint32("srt_dev.sock_type", "Socket Type", base.DEC)
+fields.ency_fld = ProtoField.uint16("srt_dev.ency_fld", "Encryption Field", base.DEC)
+fields.ext_fld = ProtoField.uint16("srt_dev.ext_fld", "Extension Field", base.HEX)
+fields.isn = ProtoField.uint32("srt_dev.isn", "Initial packet sequence number", base.DEC)
+fields.mss = ProtoField.uint32("srt_dev.mss", "Max Packet Size", base.DEC)
+fields.fc = ProtoField.uint32("srt_dev.fc", "Maximum Flow Window Size", base.DEC)
+fields.conn_type = ProtoField.uint32("srt_dev.conn_type", "Connection Type", base.DEC)
+fields.sock_id = ProtoField.uint32("srt_dev.sock_id", "Socket ID", base.DEC)
+fields.syn_cookie = ProtoField.uint32("srt_dev.syn_cookie", "SYN cookie", base.DEC)
+fields.peer_ipaddr = ProtoField.none("srt_dev.peer_ipaddr", "Peer IP address", base.NONE)
+fields.peer_ipaddr_4 = ProtoField.ipv4("srt_dev.peer_ipaddr", "Peer IP address")
+fields.peer_ipaddr_6 = ProtoField.ipv6("srt_dev.peer_ipaddr", "Peer IP address")
+local ext_type_select = {
+	[-1] = "SRT_CMD_NONE",
+	[0] = "none",
```

Comment:
This is `SRT_CMD_REJECT`, but this code should never occur in the protocol. If it happens, it's an internal error.